src/app/code/Komoju/Payments/Controller/HostedPage/Redirect.php

```diff
@@ -144,7 +144,7 @@ private function getHostedPageParams()
             "transaction[customer][given_name]" => $billingAddress->getFirstname(),
             "transaction[customer][family_name]" => $billingAddress->getLastname(),
             "transaction[customer][phone]" => $billingAddress->getTelephone(),
-            "transaction[tax]" => $order->getTaxAmount(),
```

Comment:
@jdgc since we're now removing the tax amount from the request do we need to change `"transaction[amount]"` to include the tax value?

Reply:
Unless I'm misunderstanding `$order->getGrandTotal()` should already be tax inclusive

Reply:
sounds like it was a bug then (since it would have been double including the tax value) 😅 . Code LGTM I just want to test locally to make sure it works as intended 😄